### PR TITLE
[Docs] Add code blocks for down and stop in Quickstart

### PR DESCRIPTION
**Pull request by romilbhardwaj**
Our quickstart uses codeblocks to visually indicate important commands that the user should know/run. Example:

image

However, in the `Stop/terminate a cluster` section, we use inline code, which is easy to miss:
image

This PR changes the down/stop commands to be code blocks:
image

- [x] Rendered locally

### CODE REVIEW

File: docs/source/getting-started/quickstart.rst [hunk 1]
```diff
@@ -164,8 +164,17 @@ For uploading files to the cluster, see :ref:`Syncing Code and Artifacts <sync-c
 Stop/terminate a cluster
 =========================
 
-When you are done, run :code:`sky stop mycluster` to stop the cluster. To
-terminate a cluster instead, run :code:`sky down mycluster`.
+When you are done, stop the cluster with:
```
Comment (concretevitamin) [inline docs/source/getting-started/quickstart.rst:167]: nit: add ``sky stop`` here as well, same as the launch example above? Personally thought this way it's more noticeable.

File: docs/source/getting-started/quickstart.rst [hunk 2]
```diff
@@ -164,8 +164,17 @@ For uploading files to the cluster, see :ref:`Syncing Code and Artifacts <sync-c
...
+
+.. code-block:: console
+
+  $ sky stop mycluster
+
+To terminate a cluster instead, run:
```
Comment (concretevitamin) [inline docs/source/getting-started/quickstart.rst:173]: ditto

**Review by concretevitamin — APPROVED** [review]
Thanks @romilbhardwaj!

File: docs/source/getting-started/quickstart.rst (hunk 1 above)
Comment (romilbhardwaj) [reply docs/source/getting-started/quickstart.rst]: Added!

**Pull request merged**